lib/lib.sh

```diff
@@ -132,7 +132,7 @@ create_ipv4_aliases() {
     local mac=$2
     local addresses
     subnet_supports_ipv4 "$iface" || return 0
-    addresses=$(get_iface_imds $mac local-ipv4s | tail -n +2)
+    addresses=$(get_iface_imds $mac local-ipv4s | tail -n +2 | sort)
```

Comment:
The description of this change mentions the use of `head`, but the code is using `tail`. Did you intend to change this to the use of `head` or should the description mention the use of `tail`?

Reply:
hah. wow. I wrote `headl` all over the place, including in my response to the original issue (#63), but meant `tail`. Let me go fix that.